feature-libs/my-account/organization/src/components/cost-center/form/cost-center-form.component.service.ts

```diff
@@ -0,0 +1,105 @@
+import { Injectable } from '@angular/core';
+import { FormGroup, Validators } from '@angular/forms';
+import { CostCenter, CostCenterService } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { FormsPersistenceService } from '../../shared/services/forms-persistence.service';
+
+// TODO:#form-persistence - move to ../services?
+/**
+ * Cost center component service is used when creating or editing
+ * a cost center.
+ *
+ * It offers a helpful set of methods when working with
+ * forms, creating and updating the cost center.
+ *
+ */
+@Injectable({ providedIn: 'root' })
+export class CostCenterFormComponentService {
+  protected formConfiguration: { [key: string]: any } = {
+    code: ['', Validators.required],
+    name: ['', Validators.required],
+    unit: null,
+    currency: null,
```

Comment:
It looks quite strange to me. The configration for currency is defined here as null, but it's also defined in it's dedicated component-service CurrencyCommonFormComponentService.

As noted in a different comment, I like the idea of defining form parts' configurations separately. We should not repeat ourselves (so we have currencyCommonForm component separated and reusable). And allow for extendability (so we have form component services).

So I have mixed feelings about this `currency: null` here.